django/forms/fields.py

```diff
@@ -532,10 +532,13 @@ class FileField(Field):
         'contradiction': _('Please either submit a file or check the clear checkbox, not both.')
     }
 
-    def __init__(self, *, max_length=None, allow_empty_file=False, **kwargs):
+    def __init__(self, *, max_length=None, allow_empty_file=False, multiple=False, **kwargs):
         self.max_length = max_length
         self.allow_empty_file = allow_empty_file
+        widget = kwargs.get('widget', self.widget)
         super().__init__(**kwargs)
+        if isinstance(widget, type):
+            self.widget.multiple = multiple
```

Comment:
Confusing bit. If superclass' __init__ changes self.widget then what?

Reply:
@melvyn-sopacua yes, it really is.
Super actually does change `self.widget` but it doesn't have a concept of `multiple` nor of any other argument passed to the widget.

I think a comment could go a long way here, what do you think?